roxctl/generate/netpol/npgLogger.go

```diff
@@ -0,0 +1,24 @@
+package netpol
+
+import (
+	"github.com/stackrox/rox/roxctl/common/logger"
+)
+
+// npgLogger wraps the common logging interface in a NP-Guard compatible logger
+// it mutes all warnings and errors as they are being returned explicitly by the library
+type npgLogger struct {
+	l logger.Logger
+}
+
+func newNpgLogger(l logger.Logger) *npgLogger {
+	return &npgLogger{
+		l: l,
+	}
+}
+
+func (nl *npgLogger) Debugf(format string, o ...interface{}) {}
+func (nl *npgLogger) Infof(format string, o ...interface{}) {
```

Comment:
Why only info is supported? I think warn and error are even more interesting

Reply:
The library returns all errors and warnings on the output and we print them at the end of the output to make it easier for the user to read.
If we pass a unmuted logger to the library, we will see the errors and warnings twice - as they occur and at the end of the execution.

Do you think the following comment is enough to explain this, or shall I improve it?

```go
// npgLogger wraps the common logging interface in a NP-Guard compatible logger
// it mutes all warnings and errors as they are being returned explicitly by the library
```

Reply:
My bad, did not read carefully.